batch_download_run.m

```diff
@@ -1,22 +1,43 @@
+% This script will download the dataset from the FIL for the block design SPM  
+% tutorial and will run the basic preprocessing, FFX and contrasts on it.
+% Results might be a bit different from those in the manual as some
+% default options are slightly different in this pipeline (e.g use of FAST
+% instead of AR(1))
+
 clear
 clc
 close all
 
-addToolboxs2Path = 1 ;
-FWHM_1 = 6;
+% Smoothing to apply
+FWHM = 6;
 
+% URL of the data set to download
 URL = 'http://www.fil.ion.ucl.ac.uk/spm/download/data/MoAEpilot/MoAEpilot.bids.zip';
 
-WD = pwd;
-addpath(genpath(WD))
+WD = pwd; % the directory with this script becomes the current directory
+addpath(genpath(WD)) % we add all the subfunctions that are in the sub directories
+
 
+%% Set options
 opt = getOption();
 
-opt.derivativesDir = fullfile(WD, 'MoAEpilot');
-opt.groups = {''};    % {'blnd', 'ctrl'};
-opt.subjects = {1};  % {[1:2], [1:2]};
+% respecify options here in case the getOption file has been modified on
```

Comment:
the getOption file might change often so by specifying this we make sure that this script won't fail in the future